src/angular/src/app/spark-docs/input-docs/input-docs.component.ts

```diff
@@ -252,6 +252,36 @@ import { Component } from '@angular/core';
           ></sprk-icon>
           <label sprkLabel>Select Box Label</label>
         </sprk-input-container>
+        <sprk-huge-input-container>
+          <select
+            id="select-huge"
+            data-id="select-huge"
+            aria-describedby="select-huge--error-container"
+            data-sprk-input="huge"
+            sprkInput
+          >
+            <option
```

Comment:
Whats the disabled, selected, hidden option for?

Reply:
@afebbraro Can you elaborate on why we needed to build it this way? @RCopeland - This is what Amber implemented for vanilla.